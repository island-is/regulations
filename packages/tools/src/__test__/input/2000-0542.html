<div align="center">
  I. KAFLI<br />
    <strong>Almenn ákvæði og skilgreiningar.</strong><br />
    1. gr.
  </div>

  <p>Reglugerð þessi gildir um kaffi, kaffikjarna og kaffibæti, sbr. skilgreiningar á þessum vörum í 2. gr. og í viðauka. Reglugerðin gildir ekki um vörur sem ætlaðar eru til útflutnings til ríkja utan Evrópska efnahagssvæðisins og ekki um leysanlegt ´café torrefacto´.<br />
  <br />
  <br />
  </p>

  <div align="center">
  2. gr.
  </div>

  <p>Merking orða í reglugerð þessari er sem hér segir:<br />
  <br />
    <em>Kaffi</em>: Fræ (baunir) kaffitrésins (<em>Coffea arabica</em>), óbrennd eða brennd, heil eða möluð, sem engin mikilvæg innihaldsefni eiginleg kaffi hafa verið fjarlægð úr. Aldinhýði skal algerlega hreinsa af fræjum og fræhýði (silfurhýði) eins vel og unnt er. Hráefni í kaffi má ekki innihalda meira en 7 g/100 g af öðrum efnum kaffitrésins en fræjum og ekki meira en 3 g af öðrum innihaldsefnum.<br />
  <br />
    <em>Kaffikjarni</em>: Vara, af mismunandi styrkleika, sem eingöngu er fengin með útdrætti með vatni úr brenndum kaffibaunum. Varan inniheldur leysanleg efni og ilmefni úr kaffinu. Einnig kunna að vera til staðar óleysanlegar olíur og snefill af öðrum óleysanlegum efnum úr kaffi. Við útdráttinn er ekki heimilt að nota aðferðir sem krefjast viðbættrar sýru eða basa. Með kaffikjarna í reglugerð þessari er átt við kaffikjarna, leysanlegan kaffikjarna, leysanlegt kaffi eða auðleysanlegt (instant) kaffiduft.<br />
  <br />
    <em>Kaffibætir</em>: Vara, af mismunandi styrkleika, sem eingöngu er fengin með útdrætti með vatni úr brenndum kaffifífli og er þá óheimilt að nota aðferðir sem krefjast viðbættrar sýru eða basa. Með kaffibæti í reglugerð þessari er átt við kaffibæti, leysanlegan kaffibæti eða auðleysanlegt (instant) kaffibætisduft.<br />
  <br />
    <em>Kaffifífill</em>: Afurð sem unnin er úr hreinsuðum, þurrkuðum og brenndum rótum <em>Cichorium intybus L</em>., sem notaðar eru til framleiðslu á drykkjarvörum en ekki til ræktunar á salati (jólasalat/hvítlaufssalat).<br />
  <br />
  <br />
  </p>

  <div align="center">
  II. KAFLI<br />
    <strong>Samsetning og merking.</strong><br />
    3. gr.
  </div>

  <p>Aðferðir til greiningar á kaffikjarna og kaffibæti skulu vera í samræmi við tilskipun Evrópusambandsins nr. 79/1066/EBE (OJ L327 24.12.79).<br />
  <br />
  <br />
  </p>

  <div align="center">
  4. gr.
  </div>

  <p>Heiti í viðauka skal eingöngu nota um vörur sem þar er getið og er skylt að nota þau í viðskiptum til að lýsa vörunni.<br />
  <br />
  <br />
  </p>

  <div align="center">
  5. gr.
  </div>

  <p>Auk ákvæða reglugerðar nr. 588/1993 um merkingu, auglýsingu og kynningu matvæla, skulu eftirfarandi upplýsingar koma fram á umbúðum vöru sem dreift er til neytenda:</p>

  <table border="0" cellspacing="0" cellpadding="0">
  <tr valign="top">
  <td width="26">a)</td>

  <td width="493">heiti vörunnar samanber viðauka. Að auki skal koma fram á hvaða formi hún er á eftirfarandi hátt, eftir því sem við á:</td>
  </tr>

  <tr valign="top">
  <td width="26"><img src="/icons/ecblank.gif" border="0" height="1" width="1" alt="" /></td>

  <td width="493">i) "massi" eða "á massaformi"</td>
  </tr>

  <tr valign="top">
  <td width="26"><img src="/icons/ecblank.gif" border="0" height="1" width="1" alt="" /></td>

  <td width="493">ii) "vökvi" eða "á vökvaformi"</td>
  </tr>

  <tr valign="top">
  <td width="26">b)</td>

  <td width="493">við heiti er heimilt að bæta hugtakinu "þykkt" þegar um er að ræða:</td>
  </tr>

  <tr valign="top">
  <td width="26"><img src="/icons/ecblank.gif" border="0" height="1" width="1" alt="" /></td>

  <td width="493">- <em>kaffikjarnalausn</em>, að því tilskildu að innihald þurrefnis úr kaffi sé meira en 25% miðað við þyngd;</td>
  </tr>

  <tr valign="top">
  <td width="26"><img src="/icons/ecblank.gif" border="0" height="1" width="1" alt="" /></td>

  <td width="493">- <em>kaffibætislausn</em>, að því tilskildu að innihald þurrefnis úr kaffifífli sé meira en 45% miðað við þyngd;</td>
  </tr>

  <tr valign="top">
  <td width="26">c)</td>

  <td width="493">hugtakið "kaffínlaust" fyrir kaffikjarna er heimilt að merkja, að því tilskildu að innihald kaffíns fari ekki yfir 0,3% miðað við þyngd þurrefnis í kjarnanum;</td>
  </tr>

  <tr valign="top">
  <td width="26">d)</td>

  <td width="493">hugtakið "kaffínlaust" fyrir kaffi er heimilt að merkja, að því tilskildu að innihald kaffíns fari ekki yfir 0,1% miðað við þyngd;</td>
  </tr>

  <tr valign="top">
  <td width="26">e)</td>

  <td width="493">hugtakið "sykurbrennt" ef kjarninn er unninn úr sykurbrenndu hráefni og hugtökin "með sykri", "varið skemmdum með sykri" eða "með viðbættum sykri" er heimilt að merkja á umbúðir ef sykri hefur verið bætt í hráefnið eftir brennslu, þegar um kaffikjarnalausn og lausn af kaffibæti er að ræða. Þegar aðrar sykurtegundir en súkrósi eru notaðar skal þess getið í staðinn fyrir "sykur";</td>
  </tr>

  <tr valign="top">
  <td width="26">f)</td>

  <td width="493">fyrir kaffikjarnamassa og kaffikjarnalausn skal lágmarksinnihald kaffiþurrefnis gefið upp sem hundraðshluti af þyngd fullunninnar vöru. Hið sama gildir um kaffibætismassa og kaffibætislausn.</td>
  </tr>

  <tr valign="top">
  <td width="26"><img src="/icons/ecblank.gif" border="0" height="1" width="1" alt="" /></td>

  <td width="493"><img src="/icons/ecblank.gif" border="0" height="1" width="1" alt="" /></td>
  </tr>
  </table>

  <div style="margin-left: 2em">
  <br />
    Upplýsingar samkvæmt liðum a-f skal merkja á sama sjónsviði og merking nettóþyngdar og geymsluþols.<br />
  <br />
  <br />


  <div align="center">
  III. KAFLI<br />
    <strong>Ýmis ákvæði og gildistaka.</strong><br />
    6. gr.
  </div>
  Innlendur framleiðandi eða innflytjandi er ábyrgur fyrir því að vörur sem hér eru á markaði og um getur í viðauka séu í samræmi við ákvæði þessarar reglugerðar.<br />
  <br />
  <br />


  <div align="center">
  7. gr.
  </div>
  Eftirlit með því að ákvæðum þessarar reglugerðar sé framfylgt skal vera í samræmi við IV. kafla reglugerðar nr. 522/1994 um hollustuhætti við framleiðslu og dreifingu matvæla.<br />
  <br />
  <br />


  <div align="center">
  8. gr.
  </div>
  Með brot gegn reglugerð þessari skal farið samkvæmt lögum nr. 93/1995 um matvæli og lögum nr. 7/1998 um hollustuhætti og mengunarvarnir, með síðari breytingum, nema þyngri refsing liggi við samkvæmt öðrum lögum. Með mál sem rísa út af brotum gegn reglugerð þessari skal farið að hætti opinberra mála.<br />
  <br />
  <br />


  <div align="center">
  9. gr.
  </div>
  Reglugerð þessi er sett með stoð í 16. og 18. gr. laga nr. 93/1995 um matvæli. Einnig var höfð hliðsjón af ákvæðum samningsins um Evrópskt efnahagssvæði, sem vísað er til í II. viðauka XII. kafla, 22. tölul., tilskipun 79/1066/EBE og tilskipun 99/4/EB. Reglugerðin öðlast gildi við birtingu.<br />
  <br />
    Við gildistöku reglugerðar þessarar fellur úr gildi reglugerð nr. 447/1994 um kaffi, kaffikjarna og kaffibæti.<br />
  <br />
  <br />


  <div align="center">
  <em>Ákvæði til bráðabirgða.</em>
  </div>
  Fyrir vörutegundir sem eru hér á markaði við gildistöku þessarar reglugerðar og sem ekki eru í samræmi við ákvæði hennar, er veittur frestur til 30. september 2001 til að koma á nauðsynlegum breytingum. Hafi breytingar til samræmis við ákvæði reglugerðarinnar ekki verið gerðar að þeim tíma liðnum er dreifing vörunnar óheimil.<br />
  <br />
  <br />


  <div align="center">
  <em>Umhverfisráðuneytinu, 19. júlí 2000.</em>
  </div>
  <br />


  <div align="center">
  F. h. r.<br />
    <strong>Ingimar Sigurðsson.</strong>
  </div>
  </div>

  <div align="right">
  <em>Sigríður Stefánsdóttir.</em>
  </div>

  <div style="margin-left: 2em">
  <br />
  <br />


  <div align="center">
  VIÐAUKI<br />
    <strong>Skilgreining á framleiðsluvörum.</strong>
  </div>
  </div>
  <br />


  <table border="0" cellspacing="0" cellpadding="0">
  <tr valign="top">
  <td width="26"><strong>1.</strong></td>

  <td width="493"><strong>Kaffikjarni:</strong></td>
  </tr>

  <tr valign="top">
  <td width="26"><img src="/icons/ecblank.gif" border="0" height="1" width="1" alt="" /></td>

  <td width="493">a) <em>Þurrkaður kaffikjarni</em></td>
  </tr>

  <tr valign="top">
  <td width="26"><img src="/icons/ecblank.gif" border="0" height="1" width="1" alt="" /></td>

  <td width="493">kaffikjarni á föstu formi sem inniheldur ekki minna en 95% af kaffiþurrefni miðað við þyngd.</td>
  </tr>

  <tr valign="top">
  <td width="26"><img src="/icons/ecblank.gif" border="0" height="1" width="1" alt="" /></td>

  <td width="493">Þessi vara má ekki innihalda önnur efni en þau sem fengin eru með útdrætti hennar.</td>
  </tr>

  <tr valign="top">
  <td width="26"><img src="/icons/ecblank.gif" border="0" height="1" width="1" alt="" /></td>

  <td width="493">b) <em>Kaffikjarnamassi</em></td>
  </tr>

  <tr valign="top">
  <td width="26"><img src="/icons/ecblank.gif" border="0" height="1" width="1" alt="" /></td>

  <td width="493">kaffikjarni í formi massa, sem inniheldur 70-85% af kaffiþurrefni miðað við þyngd.</td>
  </tr>

  <tr valign="top">
  <td width="26"><img src="/icons/ecblank.gif" border="0" height="1" width="1" alt="" /></td>

  <td width="493">Þessi vara má ekki innihalda önnur efni en þau sem fengin eru með útdrætti hennar.</td>
  </tr>

  <tr valign="top">
  <td width="26"><img src="/icons/ecblank.gif" border="0" height="1" width="1" alt="" /></td>

  <td width="493">c) <em>Kaffikjarnalausn</em></td>
  </tr>

  <tr valign="top">
  <td width="26"><img src="/icons/ecblank.gif" border="0" height="1" width="1" alt="" /></td>

  <td width="493">kaffikjarni í vökvaformi sem inniheldur 15-55% af kaffiþurrefni miðað við þyngd.</td>
  </tr>

  <tr valign="top">
  <td width="26"><img src="/icons/ecblank.gif" border="0" height="1" width="1" alt="" /></td>

  <td width="493">Kaffikjarnalausn má innihalda sykur, brenndan eða óbrenndan, að hámarki 12% miðað við þyngd.</td>
  </tr>
  </table>
  <br />


  <table border="0" cellspacing="0" cellpadding="0">
  <tr valign="top">
  <td width="26"><strong>2.</strong></td>

  <td width="491"><strong>Kaffibætir:</strong><br />
    a) <em>Þurrkaður kaffibætir</em><br />
    kaffifífilskjarni í föstu formi sem inniheldur ekki minna en 95% af kaffifífilsþurrefni miðað við þyngd.<br />
    Efni sem ekki eru unnin úr kaffifífli mega ekki fara yfir 1% miðað við þyngd.<br />
    b) <em>Kaffibætismassi</em><br />
    kaffifífilskjarni í massaformi sem inniheldur 70-85% af kaffifífilsþurrefni miðað við þyngd.<br />
    Efni sem ekki eru unnin úr kaffifífli mega ekki fara yfir 1% miðað við þyngd.<br />
    c) <em>Kaffibætislausn</em><br />
    kaffifífilskjarni í vökvaformi sem inniheldur 25-55% af kaffifífilsþurrefni miðað við þyngd.<br />
    Kaffibætislausn má innihalda sykur, brenndan eða óbrenndan, að hámarki 35% miðað við þyngd.</td>
  </tr>
  </table>
