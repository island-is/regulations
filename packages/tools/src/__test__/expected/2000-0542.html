<h2 class="chapter__title">
I.
KAFLI
<em class="chapter__name">Almenn
ákvæði
og
skilgreiningar.</em>
</h2>

<h3 class="article__title">
1.
gr.
</h3>

<p>
Reglugerð
þessi
gildir
um
kaffi,
kaffikjarna
og
kaffibæti,
sbr.
skilgreiningar
á
þessum
vörum
í
2.
gr.
og
í
viðauka.
Reglugerðin
gildir
ekki
um
vörur
sem
ætlaðar
eru
til
útflutnings
til
ríkja
utan
Evrópska
efnahagssvæðisins
og
ekki
um
leysanlegt
´café
torrefacto´.
</p>

<h3 class="article__title">
2.
gr.
</h3>

<p>
Merking
orða
í
reglugerð
þessari
er
sem
hér
segir:
</p>

<p>
<em>Kaffi</em>:
Fræ
(baunir)
kaffitrésins
(<em>Coffea
arabica</em>),
óbrennd
eða
brennd,
heil
eða
möluð,
sem
engin
mikilvæg
innihaldsefni
eiginleg
kaffi
hafa
verið
fjarlægð
úr.
Aldinhýði
skal
algerlega
hreinsa
af
fræjum
og
fræhýði
(silfurhýði)
eins
vel
og
unnt
er.
Hráefni
í
kaffi
má
ekki
innihalda
meira
en
7
g/100
g
af
öðrum
efnum
kaffitrésins
en
fræjum
og
ekki
meira
en
3
g
af
öðrum
innihaldsefnum.
</p>

<p>
<em>Kaffikjarni</em>:
Vara,
af
mismunandi
styrkleika,
sem
eingöngu
er
fengin
með
útdrætti
með
vatni
úr
brenndum
kaffibaunum.
Varan
inniheldur
leysanleg
efni
og
ilmefni
úr
kaffinu.
Einnig
kunna
að
vera
til
staðar
óleysanlegar
olíur
og
snefill
af
öðrum
óleysanlegum
efnum
úr
kaffi.
Við
útdráttinn
er
ekki
heimilt
að
nota
aðferðir
sem
krefjast
viðbættrar
sýru
eða
basa.
Með
kaffikjarna
í
reglugerð
þessari
er
átt
við
kaffikjarna,
leysanlegan
kaffikjarna,
leysanlegt
kaffi
eða
auðleysanlegt
(instant)
kaffiduft.
</p>

<p>
<em>Kaffibætir</em>:
Vara,
af
mismunandi
styrkleika,
sem
eingöngu
er
fengin
með
útdrætti
með
vatni
úr
brenndum
kaffifífli
og
er
þá
óheimilt
að
nota
aðferðir
sem
krefjast
viðbættrar
sýru
eða
basa.
Með
kaffibæti
í
reglugerð
þessari
er
átt
við
kaffibæti,
leysanlegan
kaffibæti
eða
auðleysanlegt
(instant)
kaffibætisduft.
</p>

<p>
<em>Kaffifífill</em>:
Afurð
sem
unnin
er
úr
hreinsuðum,
þurrkuðum
og
brenndum
rótum
<em>Cichorium
intybus
L</em>.,
sem
notaðar
eru
til
framleiðslu
á
drykkjarvörum
en
ekki
til
ræktunar
á
salati
(jólasalat/hvítlaufssalat).
</p>

<h2 class="chapter__title">
II.
KAFLI
<em class="chapter__name">Samsetning
og
merking.</em>
</h2>

<h3 class="article__title">
3.
gr.
</h3>

<p>
Aðferðir
til
greiningar
á
kaffikjarna
og
kaffibæti
skulu
vera
í
samræmi
við
tilskipun
Evrópusambandsins
nr.
79/1066/EBE
(OJ
L327
24.12.79).
</p>

<h3 class="article__title">
4.
gr.
</h3>

<p>
Heiti
í
viðauka
skal
eingöngu
nota
um
vörur
sem
þar
er
getið
og
er
skylt
að
nota
þau
í
viðskiptum
til
að
lýsa
vörunni.
</p>

<h3 class="article__title">
5.
gr.
</h3>

<p>
Auk
ákvæða
reglugerðar
nr.
588/1993
um
merkingu,
auglýsingu
og
kynningu
matvæla,
skulu
eftirfarandi
upplýsingar
koma
fram
á
umbúðum
vöru
sem
dreift
er
til
neytenda:
</p>

<table class="layout layout--list">

<tbody>

<tr>

<td>
a)
</td>

<td>
heiti
vörunnar
samanber
viðauka.
Að
auki
skal
koma
fram
á
hvaða
formi
hún
er
á
eftirfarandi
hátt,
eftir
því
sem
við
á:
</td>

</tr>

<tr>

<td>
</td>

<td>
i)
"massi"
eða
"á
massaformi"
</td>

</tr>

<tr>

<td>
</td>

<td>
ii)
"vökvi"
eða
"á
vökvaformi"
</td>

</tr>

<tr>

<td>
b)
</td>

<td>
við
heiti
er
heimilt
að
bæta
hugtakinu
"þykkt"
þegar
um
er
að
ræða:
</td>

</tr>

<tr>

<td>
</td>

<td>
-
<em>kaffikjarnalausn</em>,
að
því
tilskildu
að
innihald
þurrefnis
úr
kaffi
sé
meira
en
25%
miðað
við
þyngd;
</td>

</tr>

<tr>

<td>
</td>

<td>
-
<em>kaffibætislausn</em>,
að
því
tilskildu
að
innihald
þurrefnis
úr
kaffifífli
sé
meira
en
45%
miðað
við
þyngd;
</td>

</tr>

<tr>

<td>
c)
</td>

<td>
hugtakið
"kaffínlaust"
fyrir
kaffikjarna
er
heimilt
að
merkja,
að
því
tilskildu
að
innihald
kaffíns
fari
ekki
yfir
0,3%
miðað
við
þyngd
þurrefnis
í
kjarnanum;
</td>

</tr>

<tr>

<td>
d)
</td>

<td>
hugtakið
"kaffínlaust"
fyrir
kaffi
er
heimilt
að
merkja,
að
því
tilskildu
að
innihald
kaffíns
fari
ekki
yfir
0,1%
miðað
við
þyngd;
</td>

</tr>

<tr>

<td>
e)
</td>

<td>
hugtakið
"sykurbrennt"
ef
kjarninn
er
unninn
úr
sykurbrenndu
hráefni
og
hugtökin
"með
sykri",
"varið
skemmdum
með
sykri"
eða
"með
viðbættum
sykri"
er
heimilt
að
merkja
á
umbúðir
ef
sykri
hefur
verið
bætt
í
hráefnið
eftir
brennslu,
þegar
um
kaffikjarnalausn
og
lausn
af
kaffibæti
er
að
ræða.
Þegar
aðrar
sykurtegundir
en
súkrósi
eru
notaðar
skal
þess
getið
í
staðinn
fyrir
"sykur";
</td>

</tr>

<tr>

<td>
f)
</td>

<td>
fyrir
kaffikjarnamassa
og
kaffikjarnalausn
skal
lágmarksinnihald
kaffiþurrefnis
gefið
upp
sem
hundraðshluti
af
þyngd
fullunninnar
vöru.
Hið
sama
gildir
um
kaffibætismassa
og
kaffibætislausn.
</td>

</tr>

</tbody>

</table>

<p style="margin-left: 2em">
Upplýsingar
samkvæmt
liðum
a-f
skal
merkja
á
sama
sjónsviði
og
merking
nettóþyngdar
og
geymsluþols.
</p>

<h2 class="chapter__title">
III.
KAFLI
<em class="chapter__name">Ýmis
ákvæði
og
gildistaka.</em>
</h2>

<h3 class="article__title">
6.
gr.
</h3>

<p style="margin-left: 2em">
Innlendur
framleiðandi
eða
innflytjandi
er
ábyrgur
fyrir
því
að
vörur
sem
hér
eru
á
markaði
og
um
getur
í
viðauka
séu
í
samræmi
við
ákvæði
þessarar
reglugerðar.
</p>

<h3 class="article__title">
7.
gr.
</h3>

<p style="margin-left: 2em">
Eftirlit
með
því
að
ákvæðum
þessarar
reglugerðar
sé
framfylgt
skal
vera
í
samræmi
við
IV.
kafla
reglugerðar
nr.
522/1994
um
hollustuhætti
við
framleiðslu
og
dreifingu
matvæla.
</p>

<h3 class="article__title">
8.
gr.
</h3>

<p style="margin-left: 2em">
Með
brot
gegn
reglugerð
þessari
skal
farið
samkvæmt
lögum
nr.
93/1995
um
matvæli
og
lögum
nr.
7/1998
um
hollustuhætti
og
mengunarvarnir,
með
síðari
breytingum,
nema
þyngri
refsing
liggi
við
samkvæmt
öðrum
lögum.
Með
mál
sem
rísa
út
af
brotum
gegn
reglugerð
þessari
skal
farið
að
hætti
opinberra
mála.
</p>

<h3 class="article__title">
9.
gr.
</h3>

<p style="margin-left: 2em">
Reglugerð
þessi
er
sett
með
stoð
í
16.
og
18.
gr.
laga
nr.
93/1995
um
matvæli.
Einnig
var
höfð
hliðsjón
af
ákvæðum
samningsins
um
Evrópskt
efnahagssvæði,
sem
vísað
er
til
í
II.
viðauka
XII.
kafla,
22.
tölul.,
tilskipun
79/1066/EBE
og
tilskipun
99/4/EB.
Reglugerðin
öðlast
gildi
við
birtingu.
</p>

<p style="margin-left: 2em">
Við
gildistöku
reglugerðar
þessarar
fellur
úr
gildi
reglugerð
nr.
447/1994
um
kaffi,
kaffikjarna
og
kaffibæti.
</p>

<h3 class="article__title article__title--provisional">
Ákvæði
til
bráðabirgða.
</h3>

<p style="margin-left: 2em">
Fyrir
vörutegundir
sem
eru
hér
á
markaði
við
gildistöku
þessarar
reglugerðar
og
sem
ekki
eru
í
samræmi
við
ákvæði
hennar,
er
veittur
frestur
til
30.
september
2001
til
að
koma
á
nauðsynlegum
breytingum.
Hafi
breytingar
til
samræmis
við
ákvæði
reglugerðarinnar
ekki
verið
gerðar
að
þeim
tíma
liðnum
er
dreifing
vörunnar
óheimil.
</p>

<p class="Dags" align="center" style="margin-left: 2em">
<em>Umhverfisráðuneytinu,
19.
júlí
2000.</em>
</p>

<p align="center" style="margin-left: 2em">
F.
h.
r.<br />
<strong>Ingimar
Sigurðsson.</strong>
</p>

<p align="right">
<em>Sigríður
Stefánsdóttir.</em>
</p>

<h2 class="chapter__title chapter__title--appendix">
VIÐAUKI
<em class="chapter__name">Skilgreining
á
framleiðsluvörum.</em>
</h2>

<table class="layout layout--list">

<tbody>

<tr>

<td>
<strong>1.</strong>
</td>

<td>
<strong>Kaffikjarni:</strong>
</td>

</tr>

<tr>

<td>
</td>

<td>
a)
<em>Þurrkaður
kaffikjarni</em>
</td>

</tr>

<tr>

<td>
</td>

<td>
kaffikjarni
á
föstu
formi
sem
inniheldur
ekki
minna
en
95%
af
kaffiþurrefni
miðað
við
þyngd.
</td>

</tr>

<tr>

<td>
</td>

<td>
Þessi
vara
má
ekki
innihalda
önnur
efni
en
þau
sem
fengin
eru
með
útdrætti
hennar.
</td>

</tr>

<tr>

<td>
</td>

<td>
b)
<em>Kaffikjarnamassi</em>
</td>

</tr>

<tr>

<td>
</td>

<td>
kaffikjarni
í
formi
massa,
sem
inniheldur
70-85%
af
kaffiþurrefni
miðað
við
þyngd.
</td>

</tr>

<tr>

<td>
</td>

<td>
Þessi
vara
má
ekki
innihalda
önnur
efni
en
þau
sem
fengin
eru
með
útdrætti
hennar.
</td>

</tr>

<tr>

<td>
</td>

<td>
c)
<em>Kaffikjarnalausn</em>
</td>

</tr>

<tr>

<td>
</td>

<td>
kaffikjarni
í
vökvaformi
sem
inniheldur
15-55%
af
kaffiþurrefni
miðað
við
þyngd.
</td>

</tr>

<tr>

<td>
</td>

<td>
Kaffikjarnalausn
má
innihalda
sykur,
brenndan
eða
óbrenndan,
að
hámarki
12%
miðað
við
þyngd.
</td>

</tr>

</tbody>

</table>

<table class="layout layout--list">

<tbody>

<tr>

<td>
<strong>2.</strong>
</td>

<td>
<strong>Kaffibætir:</strong><br />
a)
<em>Þurrkaður
kaffibætir</em><br />
kaffifífilskjarni
í
föstu
formi
sem
inniheldur
ekki
minna
en
95%
af
kaffifífilsþurrefni
miðað
við
þyngd.<br />
Efni
sem
ekki
eru
unnin
úr
kaffifífli
mega
ekki
fara
yfir
1%
miðað
við
þyngd.<br />
b)
<em>Kaffibætismassi</em><br />
kaffifífilskjarni
í
massaformi
sem
inniheldur
70-85%
af
kaffifífilsþurrefni
miðað
við
þyngd.<br />
Efni
sem
ekki
eru
unnin
úr
kaffifífli
mega
ekki
fara
yfir
1%
miðað
við
þyngd.<br />
c)
<em>Kaffibætislausn</em><br />
kaffifífilskjarni
í
vökvaformi
sem
inniheldur
25-55%
af
kaffifífilsþurrefni
miðað
við
þyngd.<br />
Kaffibætislausn
má
innihalda
sykur,
brenndan
eða
óbrenndan,
að
hámarki
35%
miðað
við
þyngd.
</td>

</tr>

</tbody>

</table>
